src/types/card-types.ts

```diff
@@ -1,4 +1,12 @@
+import { IGatsbyImageData } from 'gatsby-plugin-image'
+
 export type Action = {
   url: string
   label: string
 }
+
+export type DynamicCardImage = {
+  image: IGatsbyImageData
+  alt: string
+  isRound?: boolean
```

Comment:
suggestion: use a variant enum instead of a boolean
suggestion: add image to naming as it only applies to the image

Put together it would lead to `imageVariant: ImageVariant` with `square` being the default and `round` being an option. `circle` could then be added to force a perfect circle if needed later without the introduction of a new boolean.

Reply:
@raae That is an awesome idea! I will go ahead and implement. I do not see a dedicated folder for enums like we have for interfaces/types. Should I create an `enums` folder that sits next to the `types` folder?

Reply:
@JasonFritsche maybe put `enums` in `types` so we don't have too many folders in `/src`?

Reply:
I would add the enum to the component file. As it's not intended to be reused.

Reply:
@jtfairbank I can do that. Should we have one file called `enums.ts`, or break it up by domain? If we break it up by domain, is there a naming convention that you'd like to use?

Reply:
@JasonFritsche I agree with @raae, adding the enum in the component folder next to the component makes the most sense, as this is a component-specific enum.

Reply:
@raae @jtfairbank I've added the ImageVariant enum to `Card.tsx`.